v0.3/handbook/libraries/stdlib.md

```diff
@@ -1,26 +1,30 @@
 ---
-layout: default
-title: Standard Library
+title: "Standard Library"
 ---
 
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+
 # Standard Library
 
-## Aggregates
+##  Aggregates
 
 <table>
-
   <tr>
-    <td colspan="2">
```

Comment:
These need to be colspan 2 or they look funny. For example: https://github.com/cmontella/docs/blob/e788fcd80d5a2b9873677799b0030d0481707387/v0.3/handbook/libraries/stdlib.md